content/javascript/concepts/accessors/accessors.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'Accessors'
+Description: "Provides access to an object's properties by using the dot notation or the bracket notation."
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'OOP'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In JavaScript, object properties can be accessed using accessor known as getters. The `get` keyword is used to define a getter method for establishing custom behavior when accessing the values of object properties.
```

Comment:
A JavaScript object property can be accessed using accessors known as getters. The `get` keyword is used to define a getter method for establishing custom behavior when accessing the values of object properties.